### DISCUSSION TITLE
Any experience using Backpack with DynamoDB as the underlying database layer?

### DISCUSSION THREAD

**Discussion by stevecrossuk** [Laravel-Backpack/community-forum#708] [Q&A (Help)]
Hey Everyone,

I'm looking at utilising AWS DynamoDB as our underlying data layer in an upcoming Laravel project. I was wondering if anyone has any experience of using Backpack with DynamoDB, and if it's compatible at all?

Would appreciate any experience or advice that you have.

Thanks

**Comment by promatik** [comment]
Hi @stevecrossuk!

I've been using Backpack with different types of database, not DynamoDB, but I've used MongoDB, Redis and PostgreSQL, until now without issues.

Note that I've always used relational databases for most of the data, along with NoSQL specifically for some situations.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
